README.md

```diff
@@ -7,4 +7,6 @@ This repository contains sample applications for [ScalarDB](https://github.com/s
 - [Multi-storage Transaction Sample](multi-storage-transaction-sample/)
 - [Microservice Transaction Sample](microservice-transaction-sample/)
 - [ScalarDB GraphQL Sample](scalardb-graphql-sample/)
+- [Spring Data JDBC for ScalarDB Sample](spring-data-sample/)
+- [Spring Data JDBC for ScalarDB with Multi-storage Transactions Sample](spring-data-multi-storage-transaction-sample/)
```

Comment:
Found the index didn't have the `Spring Data JDBC` samples.

Reply:
Thanks! 🙇